docs/aws/README.md

```diff
@@ -141,13 +136,29 @@ kubectl scale deployment inflate --replicas 5
 kubectl logs -f -n karpenter $(kubectl get pods -n karpenter -l karpenter=controller -o name)
 ```
 
+You can see what EC2 instance type was added to your cluster from Karpenter with
+```bash
+kubectl get no -L "node.kubernetes.io/instance-type"
+```
+
+If you scale down the deployment replicas the instance will be terminated after 30 seconds (ttlSeconds).
+```bash
+kubectl scale deployment inflate --replicas 0
+```
+
+Or you can manually delete the node with
```

Comment:
Add a note that karpenter will inject a finalizer and terminate the instance, otherwise experienced folks won't trust this.